NEWS.md

```diff
@@ -3,6 +3,161 @@ XGBoost Change Log
 
 This file records the changes in xgboost library in reverse chronological order.
 
+## v1.7.0 (2022 Oct 20)
+
+We are excited to announce the feature packed XGBoost 1.7 release. The release note will walk through some of the major new features first, then make a summary for other improvements and language-binding-specific changes.
+
+### PySpark
+
+XGBoost 1.7 features initial support for PySpark integration. The new interface is adopted from the existing PySpark XGBoost interface developed by databricks with additional features like `QuantileDMatrix` and rapidsai plugin (GPU pipeline) support. The new Spark XGBoost Python estimators not only benefits from PySpark ml facilities for powerful distributed computing but also enjoys the rest of the Python ecosystem. Users can define a custom objective, callbacks, and metrics in Python and use them with this interface on distributed clusters. The support is labeled as experimental with more features to come in future releases. For a brief introduction please visit the tutorial on XGBoost's [document page](https://xgboost.readthedocs.io/en/latest/tutorials/spark_estimator.html). (#8355, #8344, #8335, #8284, #8271, #8283, #8250, #8231, #8219, #8245, #8217, #8200, #8173, #8172, #8145, #8117, #8131, #8088, #8082, #8085, #8066, #8068, #8067, #8020, #8385)
```

Comment:
Q: When can we remove the experimental label ?

Reply:
@WeichenXu123 I think we will have at least one release for that. I will walk through the open issues for pyspark later on and revise a roadmap for improving it after 1.7.

Reply:
@WeichenXu123 I started a new github project on pyspark https://github.com/dmlc/xgboost/projects/5 .